app/wrappers/prospect_api_wrapper.rb

```diff
@@ -0,0 +1,45 @@
+class ProspectApiWrapper
+  require 'rest-client'
+
+  API_KEY = 'a2d2142ae77ee3ff86fd6ac644e339d9'.freeze
+  BASE_URL = 'http://prospects-api.herokuapp.com/'.freeze
+  SUCCESS = 200
+
+  def self.get_prospects
+    response = get('prospects')
+
+    if success?(response)
+      prospects = JSON.parse(response.body).with_indifferent_access
+
+      prospects.fetch(:data).map do |prospect_data|
+        prospect_data_fetch = prospect_data.fetch(:attributes)
+        {
+          name: prospect_data_fetch.fetch(:name),
+          description: prospect_data_fetch.fetch(:description),
+          uid: prospect_data.fetch(:id),
+          target: calculate_target(prospect_data_fetch.fetch(:country))
+        }
+      end
+    end
+  end
+
+  private_class_method
```

Comment:
This method accepts arguments afaik... http://apidock.com/ruby/Module/private_class_method